README.md

```diff
@@ -11,28 +11,41 @@
 
 Besu is an Apache 2.0 licensed, MainNet compatible, Ethereum client written in Java.
 
+## Useful Links
+
+* [Besu Docs](http://besu.hyperledger.org/)
+* [Besu Jira](https://jira.hyperledger.org/projects/BESU/issues)
+* [Besu Wiki](https://wiki.hyperledger.org/display/BESU/Hyperledger+Besu)
+* [How to Contribute to Besu](CONTRIBUTING.md)
+
+
 ## Issues 
 
-Besu issues are tracked in [Jira](https://pegasys1.atlassian.net/secure/Dashboard.jspa?selectPageId=10117) not GitHub. 
-See our [contribution guidelines](CONTRIBUTING.md) for more detail on searching and creating issues. 
+Besu issues are tracked in [Jira](https://jira.hyperledger.org/projects/BESU/issues) not GitHub. 
+See our [contribution guidelines](CONTRIBUTING.md) for more details on searching and creating issues.
+
+If you've commented on an existing issue and have been waiting for a reply, or want to message us for any other reason, feel free to ping us on [RocketChat](https://chat.hyperledger.org/).
```

Comment:
Does calling this out make it sound like they're likely to have to wait around for a reply - should we just say something like "If you have any questions, queries or comments, RocketChat is the place to find us."